groups/payment.yml

```diff
@@ -0,0 +1 @@
+name: Payment
```

Comment:
A group makes sense here, but based on precedent in groups so far, should probably be plural - `payments`.

Reply:
```suggestion
name: Payments
```